src/api/Api.ts

```diff
@@ -98,6 +98,12 @@ export class ApiImpl implements Api {
                 ...rawResult,
                 paths: ApiImpl.decodeResult(rawResult, completeRequest.elevation),
             }
+        } else if (response.status !== 400) {
+            console.log(`route request failed, status: '${response.statusText}'`)
+            if (response.status === 500)
+                // not always true, but most of the time :)
+                throw new Error('Route calculation timed out')
+            throw new Error('Route request failed')
```

Comment:
After thinking about this again I decided to use these more user-friendly error messages. WDYT? And should we translate the error messages (already?)

Reply:
Since we have translation now, I'd say yes, if we have an appropriate translation.

I find the different paths of the else if block not easy to read. isn't it somethig along the lines like 
```
if (response.ok) {...}
else if (response.status === 500) {
    throw new Error('Route calculation timed out')
} else if (response.status === 400) {
    throw new Error('Insert your meaningful bad request message here')
} else {
    console.log(`route request failed, status: '${response.statusText}'`)
    throw new Error('Route request failed.')
}

Reply:
> if we have an appropriate translation

Everyone can [add translations here](https://docs.google.com/spreadsheets/d/10HKSFmxGVEIO92loVQetVmjXT0qpf3EA2jxuQSSYTdU/edit?pli=1#gid=0) and we can use them now, yes :)

Add text to English at least (and maybe German) and prefix the key with `web.` and then run `python3 update-translations.py`.

Reply:
I cleaned up the if/else logic and added the translations (this worked right away :+1:)